Cargo.toml

```diff
@@ -19,3 +19,39 @@ name = "inx-chronicle"
 path = "src/main.rs"
 
 [dependencies]
+actix = "0.13"
+anyhow = "1.0"
+cfg-if = "1.0"
```

Comment:
I might be wrong, but do we actually use this right now?

Reply:
Yes, it is used for mongo config.